### PR TITLE
OBJ-242 chips client stub

### PR DESCRIPTION
**Pull request by mattch1**
Stubbed client and unit test. As the method is void there is not a lot to add.

### CODE REVIEW

File: src/test/java/chips/ChipsClientTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package chips;
```
Comment (mwestacott) [inline src/test/java/chips/ChipsClientTest.java:1]: Package is wrong here. Needs to match where the class is and the ChipsClient import statement below should then not be needed.

File: src/test/java/chips/ChipsClientTest.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    private ChipsClient chipsClient;
+
+    @Mock
+    private ApiLogger apiLogger;
+
+    @Test
+    void testSendMessageToChips(){
+        // TODO OBJ-240 test sending contact data to CHIPS
+        chipsClient.sendToChips(REQUEST_ID);
+        assertEquals("test123", REQUEST_ID);
```
Comment (mwestacott) [inline src/test/java/chips/ChipsClientTest.java:30]: This assertion is pretty pointless - it's checking the unit-test code not the implementation. Appreciate there's very little at the moment in the implementation to check. Maybe this test class just isn't required at this point?

Comment (markpit) [reply src/test/java/chips/ChipsClientTest.java:30]: the real method will call apiLogger, so you could verify that apiLogger is called...

File: src/main/java/uk/gov/companieshouse/api/strikeoffobjections/chips/ChipsClient.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    private ApiLogger apiLogger;
+
+    @Autowired
+    public ChipsClient(ApiLogger apiLogger) {
+        this.apiLogger = apiLogger;
+    }
+
+    public void sendToChips(String requestId){
+        // TODO OBJ-240 add contact model object and send to CHIPS
+        Map<String, Object> logMap = new HashMap<>();
```
Comment (markpit) [inline src/main/java/uk/gov/companieshouse/api/strikeoffobjections/chips/ChipsClient.java:22]: there should be an infoContext method without a logMap, no point passing in an empty one.

File: src/test/java/chips/ChipsClientTest.java (hunk 1 above)
Comment (mattch1) [reply src/test/java/chips/ChipsClientTest.java]: Refactored

File: src/test/java/chips/ChipsClientTest.java (hunk 2 above)
Comment (mattch1) [reply src/test/java/chips/ChipsClientTest.java]: Made assertion more meaningful - please bear mind that this is a stub.

File: src/main/java/uk/gov/companieshouse/api/strikeoffobjections/chips/ChipsClient.java (hunk 3 above)
Comment (mattch1) [reply src/main/java/uk/gov/companieshouse/api/strikeoffobjections/chips/ChipsClient.java]: Removed empty log map.

**Review by markpit — APPROVED** [review]

**Comment by markpit** [comment]
There's a couple of Sonar issues to fix

**Review by markpit — APPROVED** [review]

**Review by mwestacott — APPROVED** [review]

**Pull request merged**